src/dispatch/incident/flows.py

```diff
@@ -97,6 +97,34 @@ def get_incident_participants(incident: Incident, db_session: SessionLocal):
     return individual_contacts, team_contacts
 
 
+def reactivate_incident_participants(incident: Incident, db_session: SessionLocal):
+    """Reactivates all incident participants."""
+    for participant in incident.participants:
+        incident_add_or_reactivate_participant_flow(
+            participant.individual.email, incident.id, db_session=db_session
+        )
+
+    event_service.log(
+        db_session=db_session,
+        source="Dispatch Core App",
+        description="Incident participants reactivated",
+        incident_id=incident.id,
+    )
+
+
+def inactivate_incident_participants(incident: Incident, db_session: SessionLocal):
+    """Inactivates all incident participants."""
+    for participant in incident.participants:
+        participant_flows.inactivate_participant(participant.individual.email, incident, db_session)
+
+    event_service.log(
+        db_session=db_session,
+        source="Dispatch Core App",
+        description="Incident participants inactivated",
+        incident_id=incident.id,
+    )
+
```

Comment:
It seems like these are pretty redundant without much actual logic, can they be combined and have a flag that states whether we should activate or inactivate participants?

Reply:
Yes, I could combine them. I felt it would be easier to maintain if I kept them separate and small.